raycicmd/config.go

```diff
@@ -11,6 +11,8 @@ import (
 // skipQueue is the queue name for skipping a buildkite runner job.
 const skipQueue = "~"
 
+const buildAuthorEnv = "BUILDKITE_BUILD_CREATOR_EMAIL"
```

Comment:
could you add a link to the docs about this env var, or add more info about what this email points to?